tools/ci-build/publisher/src/subcommand/yank_release.rs

```diff
@@ -18,6 +20,16 @@ use tracing::info;
 
 const MAX_CONCURRENCY: usize = 5;
 
+#[derive(Copy, Clone, Debug, ArgEnum, Eq, PartialEq, Ord, PartialOrd)]
+pub enum CrateSet {
+    /// (default) Yank all crates associated with the release.
+    All,
+    /// Yank all AWS SDK crates.
+    AllAwsSdk,
+    /// Yank generated AWS SDK crates.
+    GeneratedAwsSdk,
+}
+
```

Comment:
Just to make sure I understand correctly:
- `All`: AWS SDKs, AWS runtime crates, smithy runtime crates
- `AllAwsSdk`: AWS SDKs, AWS runtime crates
- `GeneratedAwsSdk`: AWS SDKs
Is that correct?

Reply:
That's correct.